CHANGELOG.md

```diff
@@ -42,6 +43,47 @@ This is a feature release:
     panic. This was due to a `nil` pointer, fixed with proper initialization
     (@HansK-p, @ju-popov, #878).
 
+### Upgrade considerations
+
+#### Module Name Suffix
+
+As required by the Go module system, a suffix with the new major version has been
```

Comment:
Don't repeat the librdkafka changelog for OpenSSL, just say like: "OpenSSL 3.0.x upgrade in librdkafka requires a major version bump, as some legacy ciphers need to be explicitly configured to continue working. The rest of the API remains backward compatible. See librdkafka release notes below. ". Then continue with "As required ..."

Reply:
"as some legacy ciphers need to be explicitly configured to continue working, but it is highly recommended NOT to use them."